pkg/kubelet/allocation/state/checkpoint.go

```diff
@@ -20,16 +20,14 @@ import (
 	"encoding/json"
 	"fmt"
 
-	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/kubernetes/pkg/kubelet/checkpointmanager"
 	"k8s.io/kubernetes/pkg/kubelet/checkpointmanager/checksum"
 )
 
 var _ checkpointmanager.Checkpoint = &Checkpoint{}
 
-type PodResourceAllocationInfo struct {
-	AllocationEntries map[types.UID]map[string]v1.ResourceRequirements `json:"allocationEntries,omitempty"`
+type PodResourceCheckpointInfo struct {
+	Entries PodResourceInfoMap `json:"entries,omitempty"`
```

Comment:
Note to anyone else looking at this PR:
This non-backwards compatible change to the checkpoint type is safe only because we renamed the checkpoint file(s) in https://github.com/kubernetes/kubernetes/pull/130599 (which is itself safe because nodes must be drained before upgrading). Since the checkpoint filepaths changed in v1.33, it means that when upgrading to a v1.33 Kubelet it will always start with a fresh checkpoint file, and doesn't need to worry about a failure to parse an existing checkpoint.